src/phongo_bson.c

```diff
@@ -768,6 +768,55 @@ static void php_phongo_handle_field_path_entry_for_compound_type(php_phongo_bson
 	}
 }
 
+#if PHP_VERSION_ID >= 80100
+/* Resolves an enum class and case name to a zval. On error, an exception will
+ * have been thrown and NULL will be returned.
+ *
+ * This function is modeled after php_var_unserialize_internal in php-src. */
+static zval* resolve_enum_case(zend_class_entry* ce, const char* case_name)
+{
+	zval*                return_value = NULL;
+	zend_string*         c_str;
```

Comment:
`c_str` will be uninitialized when `!(ce->ce_flags & ZEND_ACC_ENUM)`

```suggestion
	zend_string*         c_str = NULL;
```

Edit: Looking below, I realize that can't actually happen since you only call this method from with a `ce->ce_flags & ZEND_ACC_ENUM` block, but if you're going to belt and suspender it here, then you might as well do it right. :)